### PR TITLE
batctl: update to 2018.3.

### PR DESCRIPTION
**Pull request by damadmai**

### CODE REVIEW

File: srcpkgs/batctl/template [hunk 1]
```diff
@@ -9,7 +9,7 @@ makedepends="libnl3-devel"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/batctl/template]: please use spdx short identifier

**Comment by damadmai** [comment]
done

**Comment by cr6git** [comment]
@damadmai you forgot to include the MIT license.
````
$ xlint srcpkgs/batctl/template
srcpkgs/batctl/template:9: license 'MIT', but no use of vlicense
````

**Comment by damadmai** [comment]
included it now, as well as for #3250 and #3251.
Thanks for the very fast response!

File: srcpkgs/batctl/template [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 distfiles="http://downloads.open-mesh.net/batman/releases/batman-adv-${version}/${pkgname}-${version}.tar.gz"
-checksum=2472d0eb63e8c523cc8ceee4d104006808c76c78fb61a3afd43009c9a71276f9
+checksum=24e32d3fe8268ce0951903546e71b925b78b17acc8809ba899940f48a5e892e6
 
 do_build() {
 	make CC=$CC ${makejobs}
 }
 do_install() {
 	make PREFIX=/usr DESTDIR=${DESTDIR} SBINDIR=/usr/bin install
+	vlicense LICENSES/preferred/GPL-2.0
```
Comment (cr6git) [inline srcpkgs/batctl/template]: `xlint` never asked you to add the GPL-2.0 license. please remove this line.

**Pull request merged**